### PR TITLE
Stripping channel whitesapce

### LINKED ISSUE #51: Channel name should strip whitespace
A channel's name can contain whitespace on the front or back, strip that out automatically on save.

### PR DESCRIPTION
**Pull request by StruckCroissant**
fixes #51 
Added whitespace strip to channel api.py file via dict copy & edit

### CODE REVIEW

**Comment by lrnselfreliance** [comment]
@StruckCroissant I don't see the changes in this PR.  Check that you've pushed the correct branch?

**Comment by StruckCroissant** [comment]
@lrnselfreliance forgot to push local commits to remote branch. Fixed

File: modules/videos/channel/api.py [hunk 1]
```diff
@@ -49,7 +49,10 @@ def channel_post(_: Request, body: schema.ChannelPostRequest):
     if not body.directory.is_dir() and body.mkdir:
         make_media_directory(body.directory)
 
-    channel = lib.create_channel(body.__dict__)  # TODO don't use the dataclass as a dict.
+    bd_edit = body.__dict__
+    bd_edit['name'] = bd_edit['name'].strip()
+    channel = lib.create_channel(bd_edit)  # TODO don't use the dataclass as a dict.
+    print(channel)
```
Comment (lrnselfreliance) [inline modules/videos/channel/api.py:55]: Remove this print.

**Review by lrnselfreliance — CHANGES_REQUESTED** [review]
Also, modify a channel test to check that the name is stripped.  I suggest adding it here: `modules.videos.channel.test.test_api.TestVideoAPI.test_channel`

In the test, modify "Example Channel 1" to include some whitespace, then assert that it is stripped later in the test.

File: modules/videos/channel/api.py (hunk 1 above)
Comment (StruckCroissant) [reply modules/videos/channel/api.py]: fixed

**Review by lrnselfreliance — APPROVED** [review]

**Pull request merged**